README/ReleaseNotes/v624/index.md

```diff
@@ -126,6 +128,50 @@ See [Demo notebook in SWAN](https://github.com/hageboeck/rootNotebooks),
 [EPJ Web Conf. 245 (2020) 06007](https://www.epj-conferences.org/articles/epjconf/abs/2020/21/epjconf_chep2020_06007/epjconf_chep2020_06007.html),
 [arxiv:2012.02746](https://arxiv.org/abs/2012.02746).
 
+### RooBatchCompute Library
+A new library has been created with the aim of concentrating the auto-vectorizable compute functions of roofit,
+as well as all the helpers (classes, structs, helping functions) that provide this functionality. The PDFs contained in this library are highly optimized and there is currently work in progress for further optimization using CUDA and multi-threaded computations. If you use PDFs that are not part of the official Roofit, you are very well invited to request their addition in this library by submitting a ticket [here](https://github.com/root-project/root/issues/new).
+
+#### Benefiting from batch computations by overriding `evaluateSpan()`
+It is even possible to benefit from batch computations that are not yet included in the official Roofit. To do so, consult the [RooBatchCompute readme](https://github.com/root-project/root/tree/v6-24-00-patches/roofit/batchcompute).
+
+In case you have created a custom PDF which overrides `evaluateBatch()` instead, please follow these steps to update your code to the newest version:
```

Comment:
```suggestion

#### Migrating PDFs that override the deprecated `evaluateBatch()`
In case you have created a custom PDF which overrides `evaluateBatch()`, please follow these steps to update your code to the newest version:
```